sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/HealthApiClient.cs

```diff
@@ -40,7 +40,7 @@ public HealthApiClient(string endpoint, WebPubSubServiceClientOptions options =
             Argument.AssertNotNull(endpoint, nameof(endpoint));
             options ??= new WebPubSubServiceClientOptions();
 
-            ClientDiagnostics = new ClientDiagnostics(options);
+            ClientDiagnostics = new ClientDiagnostics(options, true);
```

Comment:
What about WebPubSubServiceClient -- where does their ClientDiagnostics get initialized? https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/WebPubSubServiceClient.cs#L26

Reply:
I believe it's not auto-generated https://github.com/Azure/azure-sdk-for-net/blob/068b3debe69ef448c43e7e3f50da28ceeae4e0b4/sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_extensions.cs#L161, we can, but we don't really have to change it.

Reply:
Since we are updating all the DPG libraries. I think we should manually update it.

@annelo-msft do you agree with manually updating it?

Reply:
@annelo-msft - WebPubSubServiceClient and ConfidentialLedger have been updated [here](https://github.com/Azure/azure-sdk-for-net/pull/28565/commits/c6ba6eb267f5ae3159c27636418d354907a7fac5).